# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/sirkull/ladspa/hotfixes.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

For some reason or another that 'time' does fail sometimes..

--- ./makefile.orig	2002-07-06 19:34:20.000000000 +0200
+++ ./makefile	2005-06-14 11:46:42.000000000 +0200
@@ -74,13 +72,13 @@
 	echo ; ../bin/analyseplugin -l ../plugins/sine.so ; echo
 	../bin/analyseplugin ../plugins/amp.so
 	../bin/analyseplugin ../plugins/noise.so
-	time ../bin/applyplugin	-s 1					\
+	../bin/applyplugin	-s 1					\
 				../snd/noise.wav /tmp/test.wav		\
 				../plugins/filter.so lpf 500		\
 				../plugins/filter.so lpf 500		\
 				../plugins/sine.so sine_fcaa 6000	\
 				../plugins/delay.so delay_5s 1 0.1	\
-				../plugins/amp.so amp_mono 4		\
+				../plugins/amp.so amp_mono 4
 
 targets:	$(PLUGINS) $(PROGRAMS)
 
